lib/MySQL_Protocol.cpp

```diff
@@ -618,7 +618,7 @@ bool MySQL_Protocol::generate_pkt_OK(bool send, void **ptr, unsigned int *len, u
 	_ptr[l]=0x00; l++;
 	l+=write_encoded_length(_ptr+l, affected_rows, affected_rows_len, affected_rows_prefix);
 	l+=write_encoded_length(_ptr+l, last_insert_id, last_insert_id_len, last_insert_id_prefix);
-	int16_t internal_status = status;
+	int16_t internal_status = server_status;
```

Comment:
@acampoh : probably this is a rightful change. Can you please give me more context?

Reply:
i think this was the case...

https://github.com/mysql-otp/mysql-otp/pull/117